### ISSUE TITLE
Template Builder causes the TemplatePack project to freeze.

### ISSUE THREAD

**Issue by farid-am** [ligershark/template-builder#67]
Template Builder causes the TemplatePack project to freeze when I try to change something in the project. Every time I add, delete, rename a file or every other project change. The freeze can take up to 5 minutes or more. 

When I remove the TemplateBuilderTargets everything goes back to normal. 

Can this be improved or what can I do to fix this?

**Comment by RandomlyKnighted** [comment]
@farid-am what version of TemplateBuilder are you using?

**Comment by farid-am** [comment]
I am using version 1.1.3

**Comment by sayedihashimi** [comment]
I noticed the issue with SideWaffle but I assumed that was because we had a lot of files in the project. I never tried removing the .targets file to see what the experience is. Are you saying that if you remove the .targets file adding/removing files is much quicker?

**Comment by farid-am** [comment]
That's correct. If I remove the import of TemplateBuilderTargets, then I can edit the TemplatePack project without any problems.

**Comment by webwizgordon** [comment]
I've noticed TemplateBuilder seems to do some kind of build when a file is added or deleted.

I copied one of my project template folders to a new sibling folder. After having VS freeze for a couple minutes, **obj/Debug/ls-templates** had a new .zip for the copied template as well as a new folder for it in **obj/Debug/ls-templates/ProjectTemplate**, despite me not building the solution. The new .zip and folder in **ob**j only contained the .vstemplate file and not the rest of the template files. No output appears in the Build Output window.

I'm not sure if this action is by design or a bug.

**Comment by gulbanana** [comment]
i am experiencing this issue also. the described workaround works for me.
